config/environment.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = function(environment) {
+  var results = {
+    APP: {
```

Comment:
We do not do a deep merge so the presence of `ENV.APP = {}` in the consuming app itself will make these values get lost...